### DISCUSSION TITLE
Where does image from generate_image end up?

### DISCUSSION THREAD

**Discussion by bkr1969** [fuatakgun/eufy_security#1071] [General]
I would like to have a recent image as a thumbnail for a generic camera, but using ```image.mycamera_event_image``` gives a picture almost a week old.  I called the service to generate an image, but nothing changed.  How do I access this image.  Using S330 cams.

**Comment by espcurious** [comment]
Good question. I have the same question.
Anyone has an idea how to find out where it is stored?
And in turn which URL can be used in a telegram_bot.send_photo action in an automation?
